kernel/aster-nix/src/fs/path/mount.rs

```diff
@@ -190,6 +191,43 @@ impl MountNode {
         Ok(())
     }
 
+    /// Move the process root and cwd to the new mount node tree.
+    fn move_process_root_and_cwd(&self, process: &Arc<Process>) {
+        let current = current!();
```

Comment:
Why use `current` here？ What is the relationship between `current` and `process` ?